### ISSUE TITLE
Running Feedme + Vizy causes issues

### ISSUE THREAD

**Issue by jan-dh** [verbb/vizy#211]
### Describe the bug

All my imports have the same error log in Feedme:

```
"Class "Tiptap\Marks\Table" not found" 
```

not sure if this is an issue on Feedme's end or Vizy.

### Steps to reproduce

1. Import blogposts into a Vizy field
2. use this Vizy config:

```
{
  "buttons": [
    "formatting",
    "bold",
    "italic",
    "underline",
    "strikethrough",
    "unordered-list",
    "ordered-list",
    "highlight",
    "undo",
    "redo",
    "link",
    "clear-format",
    "line-break"
  ],
  "toolbarFixed": true,
  "formatting": [
    "h2",
    "h3",
    "h4",
    "paragraph"
  ]
}
```

### Craft CMS version

4.4.7.1

### Plugin version

2.1.1

### Multi-site?

"yes"

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/vizy:"dev-craft-4 as 2.1.2"`

**Comment by engram-design** [comment]
Fixed in [2.1.3](https://github.com/verbb/vizy/releases/tag/2.1.3)
